tests/test_data/pipeline_results_files/sars_cov_2/results_unknown.json

```diff
@@ -8,26 +8,26 @@
         "NCOV_QC_PASS": "",
         "PANGOLIN_AMBIGUITY_SCORE": "",
         "PANGOLIN_CONFLICT": "",
-        "PANGOLIN_CONSTELLATION_VERSION": "",
-        "PANGOLIN_IS_DESIGNATED": "",
-        "PANGOLIN_LINEAGE": "",
+        "PANGOLIN_CONSTELLATION_VERSION": "v0.1.10",
+        "PANGOLIN_IS_DESIGNATED": "False",
+        "PANGOLIN_LINEAGE": "Unassigned",
         "PANGOLIN_NOTE": "",
-        "PANGOLIN_PANGO_DESIGNATION_VERSION": "",
-        "PANGOLIN_QC_NOTES": "",
-        "PANGOLIN_QC_STATUS": "",
+        "PANGOLIN_PANGO_DESIGNATION_VERSION": "PUSHER-v1.18",
+        "PANGOLIN_QC_NOTES": "failed to map",
+        "PANGOLIN_QC_STATUS": "fail",
         "PANGOLIN_SCORPIO_CALL": "",
         "PANGOLIN_SCORPIO_CONFLICT": "",
         "PANGOLIN_SCORPIO_NOTES": "",
         "PANGOLIN_SCORPIO_SUPPORT": "",
-        "PANGOLIN_SCORPIO_VERSION": "",
-        "PANGOLIN_VERSION": "",
+        "PANGOLIN_SCORPIO_VERSION": "0.3.17",
+        "PANGOLIN_VERSION": "4.2",
         "PRIMER_AMBIGUOUS_NUMREADS": "",
         "PRIMER_COVERAGE": "",
         "PRIMER_DETECTED": "",
         "PRIMER_INPUT": "",
         "PRIMER_NUMREADS": "",
         "PRIMER_UNIQUE_NUMREADS": "",
-        "STATUS": "Failed"
+        "STATUS": "Completed"
```

Comment:
This JSON file is related to the CSV file above. See my NOTE.

To expand, the sample `0774181d-*` is shared among `illumina`, `ont` and `unknown` test data. There are already samples mimicking the case of **QC-unrelated failure**, so I changed this sample to **QC-related** failure. Here `STATUS` represents a QC-unrelated sample status (e.g. cluster killed the job, memory issue, bug in pangolin, etc). Therefore when changing this sample to QC-related failure, the STATUS is `Completed` because the sample has actually completed, but errors are reported in the columns, in particular `PANGOLIN_QC_STATUS` is set to `false`, indicating that pangolin QC failed for this sample.

Originally, this sample was used for simulating the case of a sample interrupted which processing Pangolin, therefore its `STATUS` was `Failed` as that was not a QC-related failure.

Reply:
Ah I see, and before because it failed fast previously it would treat it as a failed, so this proves that it can now continue to process where it can fail `PANGOLIN_QC_STATUS` and it still completes the processing because `QC_STATUS` completed running, even though it wasn't a success.